test/test.cli.ts

```diff
@@ -26,34 +16,43 @@ describe('cli', function () {
   });
 
   it('should show output for failures', async () => {
-    const res = await execa('linkinator', ['test/fixtures/basic'], {
-      reject: false,
-    });
+    const res = await execa(
+      'node',
+      ['build/src/cli.js', 'test/fixtures/basic'],
```

Comment:
Have you thought about using package.json's `bin` property?

Reply:
Huh - suppose I could.  Would the idea be to just make sure we have the right value in package.json?

Reply:
Yup. I thought that's what you meant by "Only risk here is not entirely covering the way CLI is tested with the package." but maybe you meant something else.

Reply:
Nah you're right, that reduces some of the risk.  Good call!